pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -301,6 +301,45 @@ self: super: {
     dependencies = with self; [ completion-nvim nvim-treesitter ];
   });
 
+  coq_nvim = super.coq_nvim.overrideAttrs (old: {
+    passthru.python3Dependencies = ps: [
+      ps.pynvim
+      ps.pyyaml
+      (ps.buildPythonPackage {
+        pname = "pynvim_pp";
+        version = "40db0426f171d413dcc976b17c8eff0127664f23";
+        src = fetchFromGitHub ({
+          owner = "ms-jpq";
+          repo = "pynvim_pp";
+          rev = "40db0426f171d413dcc976b17c8eff0127664f23";
+          fetchSubmodules = false;
+          sha256 = "sha256-AtsoRJQcO5jDNlSydVnqcN7f4xURmH6bGFPpjsZ9Yrg=";
+        });
+        propagatedBuildInputs = [ ps.pynvim ];
+      })
+      (ps.buildPythonPackage {
+        pname = "std2";
+        version = "43635e036486296776b4ef5ac498f742ad25a691";
+        src = fetchFromGitHub ({
+          owner = "ms-jpq";
+          repo = "std2";
+          rev = "43635e036486296776b4ef5ac498f742ad25a691";
+          fetchSubmodules = false;
+          sha256 = "sha256-oWVaw7v1Sl7DkqIJOm8wEReID9MTD3XFOdpyQDXRa04=";
+        });
+        doCheck = true;
+      })
+    ];
+
+    # We need some patches so it stops complaining about not being in a venv
+    postPatch = ''
+      substituteInPlace coq/consts.py \
+        --replace "VARS = TOP_LEVEL / \".vars\"" "VARS = Path.home() / \".cache/home/vars\"";
```

Comment:
Why this line?

Reply:
Without it, coq will try to write things in the plugin folder (i.e. in the nix store) which makes it crash.

Reply:
`~/.cache/home/vars` seems like a too generic folder

What about ~`/.cache/coq_nvim/vars`?

Reply:
Indeed ! Done :)